### PR TITLE
feat(repo): `nx wing`

### PR DESCRIPTION
**Pull request by eladb**
Implement a global `nx wing` command that builds & run the Wing CLI. It's a great one-stop shop for iterating on the toolchain (sdk + compiler + cli).

Modified the `nx build` command within the SDK to only run the "compile" flow instead of "compile + test".

Extracted "workflows" and "wingsdk" sections from the contributors' handbook.

*By submitting this pull request, I confirm that my contribution is made under the terms of the 
[Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

File: docs/06-contributors/workflows.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+---
+title: Development Workflows
+id: workflows
+keywords: [Wing contributors, contributors, workflows]
+---
+
+This topic includes a description of common development workflows for thw Wing project.
```
Comment (MarkMcCulloh) [inline docs/06-contributors/workflows.md:7]:
```suggestion
This topic includes a description of common development workflows for the Wing project.
```

File: docs/06-contributors/workflows.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+## Environment Setup
+
+Here is a list of minimal tools you should install to build the Wing repo in your development
+environment:
+
+* [Node.js] version 18.x or above (we recommend [volta])
```
Comment (MarkMcCulloh) [inline docs/06-contributors/workflows.md:14]: While you're moving this, could you add a note that we require npm v8 (not v9) to build as well?

File: docs/06-contributors/workflows.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+[Rust]: https://www.rust-lang.org/tools/install
+[AWS CLI]: https://aws.amazon.com/cli/
+[Terraform CLI]: https://learn.hashicorp.com/terraform/getting-started/install.html
+[volta]: https://volta.sh
+
+## Development iteration
+
+The `nx wing` command can be executed from anywhere in the repository in order to build and run the
+compiler, SDK and the Wing CLI. Nx is configured to make sure only the changed components are built
+every time.
```
Comment (MarkMcCulloh) [inline docs/06-contributors/workflows.md:45]: nx runs the commands of the project you're currently it. It's expected that your current directory is the one you're currently concerned with.
```suggestion
The `nx wing` command can be executed from the root of the repository in order to build and run the
compiler, SDK and the Wing CLI. Nx is configured to make sure only the changed components are built
every time.
```

---

Alternatively:

```suggestion
The `nx wing winglang` command can be executed from anywhere in the repository in order to build and run the
compiler, SDK and the Wing CLI. Nx is configured to make sure only the changed components are built
every time.
```

nx [target] [project] can be run anywhere. Changing it back to `nx dev winglang` wouldn't look too bad.

**Review by MarkMcCulloh — APPROVED** [review]

File: docs/06-contributors/workflows.md (hunk 2 above)
Comment (eladb) [reply docs/06-contributors/workflows.md:14]: Is there an issue that tracks this issue? (so I can reference, and also understand why we are limited to v8)

Comment (MarkMcCulloh) [reply docs/06-contributors/workflows.md:14]: closest is https://github.com/winglang/wing/issues/1103

It has come up only in slack before otherwise

Comment (eladb) [reply docs/06-contributors/workflows.md:14]: Thanks!

**Pull request merged**